templates/pipelines/common-app-ci.yaml

```diff
@@ -93,25 +103,21 @@ stages:
                 Write-Host "##vso[build.updatebuildnumber]$(GetAppVersion.appVersion)-$(RunDate)-$(Revision)"
                 Write-Host "##vso[task.setvariable variable=appVersion]$(GetAppVersion.appVersion)"
 
-          - task: AzurePowerShell@5
+          - task: PowerShell@2
             condition:  |
               and(
                   eq(variables['appVersion'], variables['initialVersion']),
                   eq(variables.IsDefaultBranchBuild, true)   
                   )
-            displayName: Setup Azure Pipelines Github App            
-            name: SetupAzurePipelinesApp  
+            displayName: Setup Azure Pipelines Github App
+            name: SetupAzurePipelinesApp
             inputs:
-              azureSubscription: ${{ parameters.sonarConfig.keyVaultServiceConnection }}
-              ScriptType: filePath
-              ScriptPath: $(Agent.BuildDirectory)/s/ADO-Pipeline-Common/templates/powershell/build/Add-RepositoryToGitHubAppInstallation.ps1
-              ScriptArguments: > 
-                  -KeyVaultName ${{ parameters.sonarConfig.keyVaultName }}
-                  -KeyVaultSecretName "ADP-PLATFORM-GITHUB-PatToken"
+              filePath: $(Agent.BuildDirectory)/s/ADO-Pipeline-Common/templates/powershell/build/Add-RepositoryToGitHubAppInstallation.ps1
```

Comment:
I think its one time setup, can it be part of repro creation process?

Reply:
It can be moved to API but even from Pipeline the step would run only once as the condition is set to check if the version is 0.1.0 which is default when repo is scaffolded.

@DaRoszko - do we want to move this to API?

Reply:
not now, we can later.